dev-qt/qt3d/qt3d-5.15.6.9999.ebuild

```diff
@@ -3,14 +3,14 @@
 
 EAPI=8
 
+if [[ ${PV} != *9999* ]]; then
+	QT5_KDEPATCHSET_REV=1
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+fi
```

Comment:
Not against this change, but I'm not sure I understand the reason (commenting on a random place, the comment refers to the whole commit)

Reply:
Reason is less churn when spinning new releases from live ebuilds. Packages currently using a QT5_KDEPATCHSET_REV are highly unlikely to not do so on subsequent release(s). The only module with a single patch right now is Qt5Tools, and it will only be part of a qt-5.15.x-lts tag after 2023-06-16 at least.
This is similar to KEYWORDS that are kept in sync as well (except that we won't keep actual value in sync but let it remain set to 1).